Changes

```diff
@@ -475,6 +475,10 @@ Working version
 - GPR#1538: Make pattern matching compilation more robust to ill-typed columns
   (Gabriel Scherer and Thomas Refis, review by Luc Maranget)
 
+- MPR#7760: Exact selecton of lexing engine, or "Segfault in
+  ocamllex-generated code using 'shortest'
+  (Luc Maranget, reported by def (?), review by (?))
```

Comment:
The reporter is Stephen Dolan (see [MPR#7760](https://caml.inria.fr/mantis/view.php?id=7760)). If you consider that the analysis work that Frédéric Bour ("def") did help you write your fix, you can credit him with "Luc Maranget, Frédéric Bour, report by Stephen Dolan".

Reply:
(Also, the change entry could have the GPR number as well: `MPR#7760, GPR#1713`.)

Reply:
Also, you chose the wrong category for Changes. You are in the "4.06 maintenance branch" which should in fact be marked "4.06.1" (I'll fix this), you should go in the "OCaml 4.07" part, just below (by numeric MPR ordering):

```
- MPR#7751, GPR#1657: The toplevel prints some concrete types as abstract
  (Jacques Garrigue, report by Matej Kosik)
```

Reply:
Ok I change Change again.